tox.ini

```
@@ -2,6 +2,7 @@
 envlist =
     py37,
     py38,
+    py310,
```

Please, add the environment `py39` to the list of environments (we missed this change in #190)

```suggestion
    py39,
    py310,
```